src/lib/structures/Command.ts

```diff
@@ -489,6 +489,8 @@ export namespace ChatInputCommand {
 	export type Context = AliasPiece.Context;
 	export type RunInTypes = CommandOptionsRunType;
 	export type RunContext = ChatInputCommandContext;
+	export type Interaction = CommandInteraction;
+	export type Registry = ApplicationCommandRegistry;
```

Comment:
These should also go on the base namespace of Command (so you don't have to import both, especially since ChatInputCommand is just a type, not a class)

Reply:
Oh, brainfart moment haha